Acx2015 new config docs

acx2015 opened the pull request:
In this commit I changed the file `doc/source/old_sections/ad_inst_ta1.rst` to reflect changes in the new `config` script (please refer to [PR#103](https://github.com/trustedanalytics/atk/pull/103)).  This file needs several known changes still, as well as the standard code review:
- will need to be moved to its proper final location
- will need to be polished by Georgia or Mark
- the section on the template file parameters needs to be either updated or removed, since I'm not sure whether we are still going to provide a template file for power users to work from when they create custom `user.conf` files.

acx2015 committed:
WIP - change docs for new ATK config (post-install) script

acx2015 committed:
WIP: Updated old 'config' script section

Changed the file `doc/source/old_sections/ad_inst_ta1.rst` to reflect
changes in the config script.  This is a WIP in need of being polished by a
tech writer and moved into the right place in the `docs` directory
structure.

acx2015 commented:
Re the last point in my PR comment above: per Rene, we _are_ keeping the template file exactly as before for the time being, and we don't need to change anything in the docs referring to that file.

ashahba commented on doc/source/old_sections/ad_inst_ta1.rst:
```diff
@@ -286,7 +286,7 @@ Installing |TOOLKIT|
------------------------------
+-
 
 Install |PACKAGE| Python REST server and its dependencies.
```
Python REST server is not correct.
It should be |PACKAGE| REST server.

ashahba commented on doc/source/old_sections/ad_inst_ta1.rst:
```diff
@@ -298,7 +298,7 @@ with the HDFS name node.
     $ sudo yum -y install trustedanalytics-rest-server
```
The package  name is just "trustedanalytics"

ashahba commented on doc/source/old_sections/ad_inst_ta1.rst:
```diff
@@ -298,7 +298,7 @@ with the HDFS name node.
----------------------------
+-
 
 The |PACKAGE| spark dependencies package needs to be installed on every node
```
This section  needs update.
There are no more spark dependencies.
It was removed a while ago.

ashahba commented on doc/source/old_sections/ad_inst_ta1.rst:
What happens if they answer "no"? It should only be "local" otherwise.

ashahba commented on doc/source/old_sections/ad_inst_ta1.rst:
```diff
@@ -573,7 +642,7 @@ The configuration script configures postgresql automatically.
     single: H2
 
 H2
```
Is H2 still supported and working? I don't think anybody is testing it.

acx2015 committed:
Revert "WIP - change docs for new ATK config (post-install) script"

This reverts commit 6f04222d2a00bd58e543268438310d56649e5a44.

acx2015 committed:
Revert "WIP: Updated old 'config' script section"

This reverts commit 05622234f7ac1aa1f3e9e7a555d7e4a65c74c173.

acx2015 committed:
Remove obsolete docs describing old config script

Per Chris, remove documentation of the previous version of the
post-install "config.py" script.  Documentation of the new version
will live in the Enterprise Wiki.

acx2015 commented:
PR updated to remove documentation of the previous version of the `config` script.  New docs will live in the Secure Wiki.

acx2015 committed:
Removed manual configuration section per Rene

acx2015 commented:
PR updated _again_ to remove remaining documentation on manual configuration via the `application.conf` file (replaced by `user.conf` in the new version).  These docs will now live in the Secure Wiki.
